locksmith/src/routes/v2/ticket.ts

```diff
@@ -39,4 +42,14 @@ router.post(
   }
 )
 
+router.get(
+  '/:network/:lockAddress/:tokenId/qr',
```

Comment:
is `tokenId` synonymous to `keyId` here?

Reply:
Good point. I am changing it in other places too.